buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -66,10 +67,10 @@ public void apply(Project project) {
 
         project.getRootProject().getPluginManager().apply(ReaperPlugin.class);
 
-        ReaperService reaper = project.getRootProject().getExtensions().getByType(ReaperService.class);
+        var reaperServiceProvider = project.getRootProject().getExtensions().getByType(ReaperExtension.class).getReaperService();
```

Comment:
I have mixed feelings on using `var` mixed with explicit types.

Reply:
fixed. mixing them is probably bad indeed. I'll move forward using them more though as it makes things soo convenient and more readable